scripts/environment.sh

```diff
@@ -1,16 +1,40 @@
+#!/bin/bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftCrypto open source project
+##
+## Copyright (c) 2019 Apple Inc. and the SwiftCrypto project authors
```

Comment:
This comment should be updated to be 2020.

Reply:
fixed, also fixed in script I added yesterday `scripts/generate_boilerplate_files_with_gyb.sh`

I also updated reference to `CONTRIBUTORS.txt`, which is now `CONTRIBUTORS.md`